src/util.cpp

```diff
@@ -90,6 +90,8 @@ const int64_t nStartupTime = GetTime();
 //Dash only features
 bool fMasternodeMode = false;
 bool fDisableGovernance = false;
+std::string gCoinJoinName = "CoinJoin";
```

Comment:
Thanks for this PR!  Does it make sense to use the `const` keyword here?

Reply:
Yes, this should be const.